src/NewProject/ProjectManager/project_manager.cpp

```diff
@@ -1899,6 +1821,32 @@ QString ProjectManager::ProjectVersion(const QString& filename) {
   return QString();
 }
 
+int ProjectManager::CreateAndAddFile(const QString& suffix,
+                                     const QString& filename,
+                                     const QString& filenameAdd,
+                                     bool copyFile) {
+  int ret{-1};
+  if (!suffix.compare("v", Qt::CaseInsensitive)) {
```

Comment:
Please add support for .svh (SystemVerilog Header) format everywhere, it is the same nature as .vh (Verilog Header)

Reply:
Done